Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.py

```diff
@@ -366,8 +366,6 @@ def checkpoint_add_host_command(client: Client, name, ip_address, ignore_warning
     name = argToList(name)
     ip_address = argToList(ip_address)
     groups = argToList(groups)
-    ignore_warnings = argToBool(ignore_warnings)
```

Comment:
argToBool is not defined error occurred. Please advise how to deal with this.

Reply:
If you are getting this error while working on your XSOAR instance is because you have an outdated `Base` pack, let's update it. If you get it while running the script locally it because your CommonServerPython is not up to date.

Reply:
Base pack is now 1.10.27 358734 with warnings "Pack with ID 'DrawRelatedIncidentsCanvas' and name 'DrawRelatedIncidentsCanvas' is not comatpbile with environment version and will be ignored", but still "Failed to execute checkpoint-host-delete command. Error: name 'argToBool' is not defined" error persists. How do I update CommonServerPython?